app/src/componentLibrary/toggle/toggle.scss

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.toggle {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  font-family: $FONT-ROBOTO-REGULAR;
+  font-size: 13px;
+  line-height: 20px;
+  color: $COLOR--almost-black;
+  cursor: pointer;
+}
+
+.children-container {
+  margin-left: 40px;
+}
+
+.slider {
+  background-color: $COLOR--e-200;
+  bottom: 0;
+  cursor: pointer;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: 0.4s;
+  border-radius: 10px;
+  width: 32px;
+  height: 20px;
+  &:before {
+    position: absolute;
+    top: 2px;
+    left: 2px;
+    content: '';
+    width: 16px;
+    height: 16px;
+    background-color: $COLOR--white-two;
+    transition: 0.4s;
+  }
+}
+
+.round:before {
+  border-radius: 50%;
+}
+
+.input {
+  display: none;
+
+  &:checked + .slider {
+    background-color: $COLOR--topaz-2;
+  }
+  &:checked + .slider:before {
+    transform: translateX(12px);
+  }
+  &:hover + .slider {
+    background-color: $COLOR--e-400;
+  }
+  &:checked:hover + .slider {
+    background-color: $COLOR--topaz-hover-2;
+  }
+}
+
+.disabled {
+  opacity: 0.3;
+  pointer-events: none;
```

Comment:
Please ask UX team should toggle support title attribute or not. To avoid problem in the future cuz title attribute and pointer-events: none; don't work together